aws/provider.go

```diff
@@ -581,6 +581,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_ssm_patch_group":                              resourceAwsSsmPatchGroup(),
 			"aws_ssm_parameter":                                resourceAwsSsmParameter(),
 			"aws_ssm_resource_data_sync":                       resourceAwsSsmResourceDataSync(),
+			"aws_storagegateway_gateway_activation_key":        resourceAwsStorageGatewayGatewayActivationKey(),
```

Comment:
Sorry meant to reply to this directly as well -- we're trying to keep the resource/data source names as follows: `aws_SERVICE_THING`. In this case, the service is `storagegateway` to match the SDK and the thing is a gateway activation key (in case they introduce some other "activation key"s) which is used specifically for activating gateways via [ActivateGateway](https://docs.aws.amazon.com/sdk-for-go/api/service/storagegateway/#ActivateGatewayInput).